pkg/apis/extensions/v1beta1/types.go

```diff
@@ -907,6 +907,9 @@ type ReplicaSetStatus struct {
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/replication-controller.md#what-is-a-replication-controller
 	Replicas int32 `json:"replicas"`
 
+	// FullyLabeledReplicas is the number of pods that have labels matching the labels of the pod template of the replicaset.
```

Comment:
We generate user documentation from these comments. I prefer not to reference the field name at all, but if you do, please use the json field name, not the Go field name.

If you have spare time, I'd appreciate an update to the doc generator to automatically do this substitution. :-)